<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySQLInfo</class>
 <widget class="QWidget" name="MySQLInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>主机地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string> 用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>   密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="saveBtn">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
